src/constants/country.js

```diff
@@ -262,3 +262,8 @@ export const LANG_TO_DEFAULT_COUNTRY = {
     [LANG.TR]: COUNTRY.TR,
     [LANG.ZH]: COUNTRY.CN
 };
+
+export const INSTALLMENT_COUNTRY = {
+    BR: 'BR',
```

Comment:
Can we use country constants here

Reply:
Hve grouped this in a separate object for the purpose of validation. During validation, it checks whether the locale is found in this specific group.